.github/CODEOWNERS

```diff
@@ -170,7 +170,7 @@
 # TODO(#10619): Add @bansalnitish as owner after 2020-10-31
 /tox.ini @DubeySandeep @Hudda
 /scripts/*.py @DubeySandeep
-/scripts/check_e2e_tests_are_captured_in_ci*.py @Showtim3 @DubeySandeep
+/scripts/check_e2e_tests_are_captured_in_ci*.py @kevintab95 @DubeySandeep
```

Comment:
is this a permanent transfer? If not, please create an issue and follow the general temporary codeowner replacement process.

Reply:
Yep, it is a permanent transfer.